tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -290,6 +290,11 @@ public function testForceRootUrl()
 		$url->forceRootUrl('https://www.bar.com');
 		$this->assertEquals('http://www.bar.com/foo/bar', $url->to('foo/bar'));
 
+		/**
+		 * Ensure trailing / is trimmed from root URL as UrlGenerator already handles this
+		 */
+		$url->forceRootUrl('http://www.foo.com/');
+		$this->assertEquals('http://www.foo.com/bar', $url->to('/bar'));
 
 		/**
```

Comment:
Just following what was present in the rest of the file. Have updated my changes.